community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf

```diff
@@ -612,3 +612,31 @@ EOD
   type        = any
   default     = false
 }
+
+variable "universe_domain" {
+  description = "Domain address for alternate API universe"
+  type        = string
+  default     = "googleapis.com"
+  nullable    = false
+}
+
+variable "endpoint_versions" {
+  description = "Version of the API to use (The compute service is the only API currently supported)"
+  type = object({
+    compute = string
```

Comment:
nit.
```hcl
compute = optional(string)
```

Reply:
I don't want this to be optional, and set it to default to `beta` which is what slurm-gcp was using by default for compute.